app/components/CarouselCard.tsx

```diff
@@ -51,7 +97,7 @@ type LinkProps = {
   openLink: string | ((event: GestureResponderEvent) => void)
 }
 
-const Link: VFC<LinkProps> = ({ openLink, text }) => {
+const Link: React.FunctionComponent<LinkProps> = ({ openLink, text }) => {
```

Comment:
[TOL] should `Link` be its own `component`?

Reply:
It kind of is already with the `<Button />` preset `link`. This is just something specific to the `<CarouselCard />`.